src/styles/calendar.scss

```diff
@@ -0,0 +1,52 @@
+.calendar {
+  padding: 10px;
+  display: flex;
+  gap: $gap;
+  flex-wrap: wrap;
+  width: calc(7 * ($cell-size + $gap));
```

Comment:
The variable `$cell-size` is used here but is not defined within this file. Ensure that `$cell-size` is defined either in this file or imported from another SCSS file.